.github/workflows/release.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - run: npm run typecheck
       - run: make package
       - run: 'echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc'
-      - run: 'npm publish ./pkg --tag latest'
+      - run: 'npm publish ./pkg --access=public --tag latest'
```

Comment:
This is fine, but in general isn’t needed after the first publish

Reply:
only added it temporarily so the package will publish. going to remove it (and the commented line below) before merging